dataprofiler/profilers/profiler_options.py

```diff
@@ -376,6 +376,43 @@ def _validate_helper(self, variable_path='IntOptions'):
         return super()._validate_helper(variable_path) 
 
 
+class PrecisionOptions(BooleanOption):
+    
+    def __init__(self, is_enabled=True, sample_ratio=None):
```

Comment:
should this instead have the default instead of `None`?

Reply:
None is the correct default - see the rest of the section it is used.

Basically, if none the default mechanism is used max(20k, len(data)*0.05). If it's set by the user it'll use exactly what the user defines: len(data)*0.05

Reply:
I see, so sample_ratio doesn't have a cap

Reply:
The validation function checks its between 0 and 1, but by default it'll use `max(20k, len(data)*0.05)`